public/usage-examples/graphics/option_rotate_bmp-1-example-oop.cs

```diff
@@ -0,0 +1,56 @@
+﻿using SplashKitSDK;
+
+namespace OptionRotateBmpExample
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Set the frame rate to 60 frames per second
+            const int fps = 60;
+
+            // Open a window with dimensions 800 x 600
+            SplashKit.OpenWindow("Usage Example - Option Rotate Bmp", 800, 600);
+
+            // Create a bitmap with dimensions 300 x 300
+            Bitmap bmp = SplashKit.CreateBitmap("box", 300, 300);
+
+            // Draw a red box on the bitmap
+            Quad rect = SplashKit.QuadFrom(0, 0, 300, 0, 0, 300, 300, 300);
+            SplashKit.FillQuadOnBitmap(bmp, Color.Red, rect);
+
+            // Initialize a float to hold the angle of the rotated bitmap
+            float angle = 0;
+
+            // Loop until the user exits
+            while (!SplashKit.QuitRequested())
```

Comment:
with the window changes as suggested this could become 
```suggestion
            while (!wind.CloseRequested)
```